src/content/docs/en/upgrade-astro.mdx

```diff
@@ -180,3 +180,24 @@ A major release may also include some non-breaking changes and improvements that
   Once these features are out of their experimental period, they will follow the normal semantic versioning contract.
 
 - **Improvements to the documentation** (e.g. reference and error messages). They are built from source for the `docs` repository. This allows Astro to quickly update docs fixes and improvements in the cases where documentation source content is stored in the main `astro` repository.
+
+### Node.js support and upgrade policies
+
+#### Support
+
+- Astro supports the [**latest _Maintenance_ LTS** versions of Node.js](https://nodejs.org/en/about/previous-releases#release-schedule).
```

Comment:
Presumably this should also say that the current LTS is supported. Right now, this (and the line below) implies that Astro only supports Node 18, 19, and 21, but not 20.

Reply:
Good call! I wasn't sure whether this line was meant to "assure about maintenance", but if it starts as "what we support", it should be exhaustive!

Reply:
Added in [`12dc2df` (#7652)](https://github.com/withastro/docs/pull/7652/commits/12dc2dfaf77278cfb837d0234ea338f7bb891cc9)